rpm/del-fixtures-sign.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+#
+# Delete signatures of all packages recursively in target directory.
+# One RPM utility `rpmsign` must be installed.
+
+set -euo pipefail
+
+# Print a message to stdout explaining how to use this script.
+#
+# NOTE: $0 corresponds to the the script's name only in some shells and in some
+# contexts.
+show_help() {
+fmt <<EOF
+usage: del-fixtures-sign.sh <target_dir>
+
+Delete signatures of all RPM/SRPM/DRPM files recursively in <target_dir>
+
+EOF
+cat <<EOF
+<target_dir>
+    The directory from which package's signature will be removed.
+EOF
+}
+
+# Fetch target_dir from user.
+if [ "$#" -lt 1 ]; then
+    echo 1>&2 'Error: Too few arguments received.'
+    echo 1>&2
+    show_help 1>&2
+    exit 1
+elif [ "$#" -gt 1 ]; then
+    echo 1>&2 'Error: Too many arguments received.'
+    echo 1>&2
+    show_help 1>&2
+    exit 1
+else
+    target_dir="$(realpath --canonicalize-missing "${1}")"
+fi
+
+# Delete signatures from packages in target_dir recursively
+find "${target_dir}" -name "*rpm" -exec rpmsign --delsign {} \;
```

Comment:
`find`'s `-exec` argument frequently masks non-zero return values. Use `xargs` instead. It's better at complaining when thing go wrong. Also, `*rpm` will catch files such as `not_an_rpm`. Use `\( -name 'pattern1' -o -name 'pattern2' \)`.

```bash
find "${target_dir}" \( -name '*.rpm' -o -name '*.drpm' -o -name '*.srpm' \) \
    -print0 | xargs -0 rpmsign --delsign
```